core/storage/statistics/gae_models.py

```diff
@@ -973,9 +973,9 @@ class PlaythroughModel(base_models.BaseModel):
     exp_id = ndb.StringProperty(indexed=True)
     # Version of the exploration.
     exp_version = ndb.IntegerProperty(indexed=True)
-    # ID of the issue.
-    issue_id = ndb.StringProperty(indexed=True)
-    # The customization args dict for the given issue_id.
+    # Type of the issue.
+    issue_type = ndb.StringProperty(indexed=True)
```

Comment:
Probably want a choices parameter too? See e.g. #4943.

Reply:
@seanlip Shouldn't we be trying to define all our issue/action types in feconf and use it from there? Is this actually required?

Reply:
In general the more validation the better. So yes.

(In fact it seems to me like you'll want to add `required=True` in a bunch of places, too.)

Reply:
Done.